src/workerd/api/js-rpc-target.c++

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017-2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#include <workerd/api/js-rpc-target.h>
+#include <workerd/api/global-scope.h>
+
+namespace workerd::api {
+
+kj::Promise<void> JsRpcTargetImpl::call(CallContext callContext) {
+  auto methodName = kj::heapString(callContext.getParams().getMethodName());
+  // TODO(now): Deserialize the arguments.
+  //    - Current demo doesn't take args, once we pass args on the client side I'll deserialize
+  //      on the server side.
+  auto serializedArgs = callContext.getParams().getSerializedArgs().getV8Serialized().asBytes();
+
+  // Try to execute the requested method.
+  try {
+    co_await ctx.run(
+        [this,
+         methodName=kj::mv(methodName),
+         serializedArgs = kj::mv(serializedArgs),
+         entrypointName = entrypointName,
+         &callContext] (Worker::Lock& lock) mutable {
+
+      auto reader = callContext.initResults(capnp::MessageSize { 4, 1 }).initResult();
+
+      auto handler = lock.getExportedHandler(entrypointName, ctx.getActor());
+      KJ_IF_MAYBE(h, handler) {
+        jsg::Lock& js = lock;
+
+        auto handle = h->self.getHandle(lock);
+        auto methodStr = jsg::v8StrIntern(lock.getIsolate(), methodName);
+        auto fnHandle = jsg::check(handle->Get(lock.getContext(), methodStr));
```

Comment:
Using the new `jsg::JsObject` should make this a bit easier.  For instance, if `h->self` was a `jsg::JsRef<jsg::JsObject>` instead, then this could be simplified to:

```cpp
auto handle = h->self.getHandle(lock);
auto fnHandle = handle.get(lock, methodName);
if (!(fnHandle.isFunction() && !fnHandle.isPrivate()) {
  // ...
```

Reply:
Apparently there's no `isPrivate`, and while `jsg::JsObject` has `hasPrivate()`, it relies on an experimental method:

```
  /**
   * Functionality for private properties.
   * This is an experimental feature, use at your own risk.
   * Note: Private properties are not inherited. Do not rely on this, since it
   * may change.
   */
  Maybe<bool> HasPrivate(Local<Context> context, Local<Private> key);
```

Also had trouble casting fnHandle to `v8::Function` :/